static/app/views/releases/list/releaseHealth/content.tsx

```diff
@@ -50,6 +52,11 @@ const Content = ({
   isTopRelease,
   getHealthData,
 }: Props) => {
+  const adoption_map = {
+    not_adopted: 'Not Adopted',
```

Comment:
should we use translations here?

this const could also be outside of this component

Reply:
Yeah, I wasn't sure where to put this. Was tempted to just have the backend return the text I wanted to display for this first MVP since I was coding both...but thought against it and just made the quick map here :P

Reply:
Okay, the backend is now sending timestamps and, in a future iteration before release, an actual front end dev will be coming in and do something better with the timestamps.